src/main/java/com/t4t/thought4thought/entities/User.java

```diff
@@ -1,20 +1,31 @@
 package com.t4t.thought4thought.entities;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 
-@Entity // This tells Hibernate to make a table out of this class
+@Entity
 public class User {
     @Id
     @GeneratedValue(strategy=GenerationType.AUTO)
     private Integer id;
 
-    private String name;
+    private String firstName;
+
+    private String lastName;
 
     private String email;
 
+    private String password;
+
+    private String passwordSalt;
+
+    private String userType;
+
+    private String interests;
+
+    private String fieldsOfStudy;
+
+    private String viewPoints;
```

Comment:
Just as above, the user may have more than one view point.

Reply:
Same thing as above with view points and field of study